console/console-init/ui/src/modules/iot-project-detail/components/AccessCredentials/AccessCredentials.tsx

```diff
@@ -54,7 +54,8 @@ const AccessCredentials: React.FunctionComponent<IAccessCredentialsProps> = ({
   };
   const EditIcon = () => (
     <SwitchWithToggle
-      id="access-credential-switch"
+      id="access-credentials-view-json-switchtoggle"
```

Comment:
i would like to suggest small change in this id.
 `id="access-credentials-view-json-switch-button"`